interfaces/Portalicious/src/app/components/import-file-dialog/import-file-dialog.component.html

```diff
@@ -38,10 +38,10 @@ <h3>
           />
 
           <p-scrollPanel
-            styleClass="h-32 w-full border border-grey-300 bg-grey-100 p-4"
+            styleClass="h-60 w-full border border-grey-300 bg-grey-100 p-4"
```

Comment:
Is there the option to set a `min-height` and a sort-of "auto-grow"-property? Or set a `height: auto`? (Potentially with a `max-height: 50dvh`, to not let it run out-of-hand...)

Reply:
Nah. Thats fine. If _really big errors_ occur, its allowed to look _really ugly_ :D